vespa/metadata.csv

```diff
@@ -0,0 +1,48 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+vespa.http.status.1xx.rate,gauge,,request,second,Number of responses with a 1xx status,0,vespa,
+vespa.http.status.2xx.rate,gauge,,request,second,Number of responses with a 2xx status,0,vespa,
+vespa.http.status.3xx.rate,gauge,,request,second,Number of responses with a 3xx status,0,vespa,
+vespa.http.status.4xx.rate,gauge,,request,second,Number of responses with a 4xx status,0,vespa,
+vespa.http.status.5xx.rate,gauge,,request,second,Number of responses with a 5xx status,0,vespa,
+vespa.jdisc.gc.ms.average,gauge,,millisecond,,Time spent in GC,0,vespa,
+vespa.mem.heap.free.average,gauge,,byte,,Free heap size,0,vespa,
+vespa.queries.rate,gauge,,query,second,Number of search queries,1,vespa,
+vespa.feed.operations.rate,gauge,,operation,second,Number of feed operations,1,vespa,
+vespa.query_latency.average,gauge,,millisecond,,Query processing time,-1,vespa,
+vespa.query_latency.95percentile,gauge,,millisecond,,95 percentile query processing time,-1,vespa,
+vespa.query_latency.99percentile,gauge,,millisecond,,99 percentile query processing time,-1,vespa,
+vespa.hits_per_query.average,gauge,,document,query,"Hits in the returned result, per query",0,vespa,
+vespa.totalhits_per_query.average,gauge,,document,query,Estimated total number of hits per query,0,vespa,
```

Comment:
```suggestion
vespa.totalhits_per_query.average,gauge,,hit,query,Estimated total number of hits per query,0,vespa,
```

Reply:
Thanks, I originally chose 'document'  because it seemed like 'hit' was mainly for caches.